app/views/users/new.html.erb

```diff
@@ -0,0 +1,53 @@
+<!-- Classic html style form
+<form accept-charset="UTF-8" action="/users" method="post">
```

Comment:
Please use `users_path` instead of `/users` for the action here. Your form should look something like:

```
<form action="users_path" method="post" accept-charset="UTF-8">
  <label for="user[username]">Username</label>
  <input name="user[username]" type="text" />
  <input
    type="hidden"
    name="authenticity_token"
    value="<%= form_authenticity_token %>"
  />
  <input type="submit" value="Submit" />
</form>
```